### DISCUSSION TITLE
Set up a special program for ARM-based Windows.

### DISCUSSION THREAD

**Discussion by yummy-henrry** [toeverything/AFFiNE#8762] [Q&A]
There is an obvious feeling of lag when ARM-based Windows runs x64-based AFFINE

**Comment by Brooooooklyn** [comment]
We are working on the Windows On Arm https://github.com/toeverything/AFFiNE/pull/8741

**Comment by yummy-henrry** [comment]
Thank you for your response!

Recently, while using a pen on my touch laptop, I have encountered occasional failures. Specifically, I can clearly see the pointer of my stylus, yet no ink appears on the canvas. Furthermore, I am interested in enhancing the finger recognition feature, as laptops compatible with pens are generally touch-sensitive as well. Additionally, I would like to integrate more commands into shortcuts. For non-native English speakers like myself, the frequent need to switch input methods to enter commands can hinder work efficiency. In my view, incorporating shortcut keys could significantly boost productivity.

PS: AFFINE's ink optimization surpasses that of OneNote, which is truly commendable!





---- Replied Message ----
| From | ***@***.***> |
| Date | 11/11/2024 19:33 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [toeverything/AFFiNE] Set up a special program for ARM-based Windows. (Discussion #8762) |

We are working on the Windows On Arm #8741
